### PR TITLE
Header and headline section

### PR DESCRIPTION
**Pull request by HabibPayenda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> Microverse assigned project
+
```
Comment (muneebulrehman) [inline README.md:6]: Consider adding a description to your project.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        <div class="fa-container">
+            <i class="fa-brands fa-twitter fa"></i>
+            <i class="fa-brands fa-linkedin-in fa"></i>
+            <i class="fa-brands fa-medium fa" ></i>
+            <i class="fa-brands fa-github fa"></i>
+            <i class="fa-solid fa-hand-peace fa"></i>
+        </div>
+    </section>
+</body>
+</html>
```
Comment (muneebulrehman) [inline index.html]: Your application is not matching the Figma template of your choosing. You need to use the same background image, same margins and padding and follow the exact layout as in the Figma template.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-top: 100px;
+}
+
+.primary-text {
+  font-family: Arial, Helvetica, sans-serif;
```
Comment (muneebulrehman) [inline styles.css]: Kindly use the same font family as used in the Figma template throughout your application.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    <script src="https://kit.fontawesome.com/105bcdf182.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <title>Hello World</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <a class="logo">Habib Payenda</a>
```
Comment (muneebulrehman) [inline index.html]: Let's give this `a` tag a `href` attribute. Every `a` tag should have one.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+            </p>
+            <a class="connect">Let's connect</a>
+        </div>
+        <div class="fa-container">
+            <i class="fa-brands fa-twitter fa"></i>
+            <i class="fa-brands fa-linkedin-in fa"></i>
+            <i class="fa-brands fa-medium fa" ></i>
+            <i class="fa-brands fa-github fa"></i>
+            <i class="fa-solid fa-hand-peace fa"></i>
+        </div>
```
Comment (muneebulrehman) [inline index.html]: Let's wrap all the icons in a `ul` tag and make the individual icons list items.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @HabibPayenda , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights


## Required Changes ♻️
_Check the comments under the review._

- [ ] Before asking for another review, add a summary to your PR. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HabibPayenda — COMMENTED** [review]
I have added the two requested sections for this project which are:
    1-header
    2-headline section

File: index.html (hunk 2 above)
Comment (HabibPayenda) [reply index.html]: Where can I get that background image from?

File: styles.css (hunk 3 above)
Comment (HabibPayenda) [reply styles.css]: I have tried to use that font family but the linter was giving me errors, what can you recommend doing?

File: index.html (hunk 4 above)
Comment (HabibPayenda) [reply index.html]: I will do it OK!

File: index.html (hunk 5 above)
Comment (HabibPayenda) [reply index.html]: OK for sure!

File: index.html (hunk 2 above)
Comment (muneebulrehman) [reply index.html]: Form the Figma template.

File: styles.css (hunk 3 above)
Comment (muneebulrehman) [reply styles.css]: You need to import the font family and then along with that you need a fallback font.

File: index.html (hunk 2 above)
Comment (HabibPayenda) [reply index.html]: it's not just one image, every element there is on it's on a separate image:
should I work with them separately or what should I do?

File: styles.css (hunk 3 above)
Comment (HabibPayenda) [reply styles.css]: I have found that font in google fonts, by importing you mean that I should include a script tag referencing it in the head part of my html.

**Review by sumairq — APPROVED** [review]
Hi @HabibPayenda , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow

## Optional suggestions
- [Optional] It would be nice to add pull request description at the beginning of the pull request. I can see that you have added description as a comment, however, it is better to include a description in the beginning. As you can see the main description is still empty. You can simply edit it by clicking on the three dots on the right and clicking on edit. Cheers 😃 

image

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**